src/js/components/AppVersionComponent.jsx

```diff
@@ -33,6 +37,14 @@ var AppVersionComponent = React.createClass({
     currentVersion: React.PropTypes.bool
   },
 
+  handleEditAppVersion: function () {
+    var appVersion = this.props.appVersion;
+    var router = this.context.router;
+    router.transitionTo(router.getCurrentPathname(), {}, {
+      modal: `editapp--${appVersion.id}--${appVersion.version}`
```

Comment:
Oh I see. It all began with `newapp`. So maybe we should change that to `new-app` too...